itunespy/__init__.py

```diff
@@ -267,6 +267,8 @@ def _get_result_list(json):
             unknown_result = result_item.ResultItem(item)
             result_list.append(unknown_result)
 
+        result_list[-1].country = country
```

Comment:
Why are you putting the country in the `-1` position here? I can't see any further usage of this.

Reply:
I think I may have accidentally mixed some things up. This line is where I set the country for result items (`result_list[-1]` being the result item that was created and added in the lines above). Unfortunately not all results from the API include a `country` (e.g. artists) which is why I set the country based on the query parameter `country`. I realize now that this is not only confusing but also breaking compatibility (because the query countries are 2-digit codes whereas the result countries are 3-digit codes). I'm gonna update the PR to address this.